general/development/tools/generator.md

```diff
@@ -42,7 +42,7 @@ Before executing the behat generators, you need a `feature` file you want to exe
 
 :::important
 
-The tool does not use `selenium` or any other browser wrapper. It reads the feature file and executes specific steps in it.
+The tool does not use `selenium` or any other browser wrapper. It reads the feature file and executes specific steps in it. You can check all the available steps going to `Site administration > Development > Create testing scenarios` and expanding the information container using the show more button.
```

Comment:
```suggestion
The tool does not use `selenium` or any other browser wrapper. It reads the feature file and executes specific steps in it. You can check all the available steps going to `Site administration > Development > Create testing scenarios` and expanding the information container using the **Show more** button.
```

Reply:
I think it would be better to display the **Show more** text starting with a capital letter (like it appears on the page) and highlight it using bold.